packages/pharos/src/components/combobox/pharos-combobox.ts

```diff
@@ -435,6 +448,13 @@ export class PharosCombobox extends ScopedRegistryMixin(FormMixin(FormElement))
     this.open = !this.open;
   }
 
+  private _normalizeString(textString: string) {
+    return textString
+      .normalize('NFKD')
+      .replace(/[\u0300-\u036f]/g, '')
+      .toLowerCase();
+  }
```

Comment:
There's also the method `localeCompare` that filters accents - testing it now and might removed the need for this method.

Reply:
`localeCompare` is for a different use case, not partial matching.

Reply:
Testing variations of this method, there are `search, sensitivity` options https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Collator/Collator#options

Reply:
The search option won't do partial matching either.

```js
"nèat".localeCompare("neat now", { usage: "search", ignorePunctuation: true, sensitivity: "base" })
```